Build.fsx

```diff
@@ -18,28 +18,47 @@ let nuGetPackages = !! (nuGetOutputFolder @@ "*.nupkg" )
 let signKeyPath = FullName "Src/AutoFixture.snk"
 let solutionsToBuild = !! "Src/All.sln"
 let processorArchitecture = environVar "PROCESSOR_ARCHITECTURE"
-
-type BuildVersionInfo = { assemblyVersion:string; fileVersion:string; infoVersion:string; nugetVersion:string }
-let calculateVersionFromGit buildNumber =
+// === vNext config ===
+// To disable vNext branch set all the variables below to None
+let vNextBranchName = Some "v4"
+// Specify sha1 of the master branch commit the vNext branch was forked from
+let vNextForkedCommitSha = Some "a177824f05e911c70aac1a459b8da5bd206cf410"
```

Comment:
@zvirja You can get that value with the `git-merge-base` command, so there's no need to hard-code it here. Here's an example:

```sh
git merge-base master v4
```

This will return the SHA-1 of the _first common parent_ between `master` and `v4`, which is 9d7bdd4. Now, you just need to wrap that in a function and you're good to go. 😉